src/core/border_router/routing_manager.hpp

```diff
@@ -296,6 +301,10 @@ class RoutingManager : public InstanceLocator
 
     TimerMilli mRouterSolicitTimer;
     uint8_t    mRouterSolicitCount;
+
+    // The timer used to delay routing policy evaluation with
+    // random delay between 0 and `kMaxRoutingPolicyDelay`.
```

Comment:
nit: I think we can consider removing this comment?

Reply:
sure, updated.